test.sh

```diff
@@ -10,9 +10,17 @@ origin_addr=localhost
 origin_port=8080
 injector_tcp_port=8005
 proxy_tcp_port=5678
+GOOGLE=172.217.7.142
```

Comment:
Why the IP? Google.com and other Google domains resolve is complicated source- and time-dependent ways, sometimes through CNAMEs. If you want google.com, I'd use that directly.